### PR TITLE
housekeeping

### PR DESCRIPTION
**Pull request by ianlowe13**
updates to comments, formatting, and deprecated feature removal for the mcp/housekeeping directory

### CODE REVIEW

File: mcp/housekeeping/multiplexed_labjack.c [hunk 1]
```diff
@@ -335,25 +344,8 @@ void mult_labjack_process_stream(ph_sock_t *m_sock, ph_iomask_t m_why, void *m_d
...
-            gainList2[7] = 1;
-            gainList2[8] = 1;
-            gainList2[9] = 1;
-            gainList2[10] = 1;
-            gainList2[11] = 0;
-            gainList2[12] = 0;
-            gainList2[13] = 0;
-        } else {
-            for (i = 0; i < state_data->num_channels; i++) {
+        for (i = 0; i < state_data->num_channels; i++) {
```
[the remark this answers is not in the document]
Comment (ianlowe13) [reply mcp/housekeeping/multiplexed_labjack.c:347]: lazy coding from when the multiplexed and non multiplexed labjacks diverged in ~2016/7.

state_number 1 was reserved for the ROX readout labjack and there was some funky business where we had to manually set these gain indices the way I did there to get it to use -1 to 1 V digitization properly. In this case that conditional never could have been entered in the multiplexed code in the first place, so I'm just cleaning up my old mess.

I believe I have also commented that out in labjack.c and just left it as an example for *how* to do it if we need to in the future. All labjacks we use are on the standard -10 to 10V range.

tl;dr - we couldn't get into that piece of logic and it was a leftover from when I split the libraries.

**Review by evanmayer — APPROVED** [review]
